lib/galaxy/tools/__init__.py

```diff
@@ -436,7 +436,7 @@ def sa_session( self ):
     @property
     def tool_version( self ):
         """Return a ToolVersion if one exists for our id"""
```

Comment:
I think now it returns a `StockLineage` object, so s/ToolVersion/StockLineage/ , right?